driver-core/src/main/java/com/datastax/driver/core/Cluster.java

```diff
@@ -490,7 +490,7 @@ public Cluster unregister(Host.StateListener listener) {
      */
     public Cluster register(LatencyTracker tracker) {
         checkNotClosed(manager);
-        boolean added = manager.trackers.add(tracker);
+        boolean added = manager.latencyTrackers.add(tracker);
```

Comment:
Do the changes in `Cluster` and `RequestHandler` really matter now that we've decided not to introduce a new tracker type?